web/srv/api_handlers.go

```diff
@@ -242,3 +246,39 @@ func (h *handler) handleApiTap(w http.ResponseWriter, req *http.Request, p httpr
 		}
 	}
 }
+
+func (h *handler) handleProfileDownload(w http.ResponseWriter, req *http.Request, params httprouter.Params) {
```

Comment:
Related to my comment about the route below, maybe this should be defined in `web/srv/handlers.go` instead?